frontend/src/components/RecruitItem.vue

```diff
@@ -10,8 +10,12 @@
         </div>
       </div>
       <div class="button-wrapper">
-        <button class="enroll-button" @click="onClickAdmission(recruitment.id)">
-          지원하기
+        <button
+          class="enroll-button button"
+          @click="onClickAdmission(recruitment.id)"
+          :disabled="!this.isRecruiting()"
+        >
+          {{ buttonLabel() }}
```

Comment:
기존 Button 컴포넌트를 사용하고 `v-bind:value="buttonLabel()"`를 사용하면 정상동작하네요!
```suggestion
        <Button
          class="enroll-button button"
          @click="onClickAdmission(recruitment.id)"
          :disabled="!this.isRecruiting()"
          v-bind:value="buttonLabel()"
        >
```

Reply:
오오오 이렇게 하니 깔끔하네요!!